src/main/java/org/broadinstitute/consent/http/db/mapper/DacWithDatasetsReducer.java

```diff
@@ -58,7 +60,7 @@ public void accumulate(Map<Integer, Dac> container, RowView rowView) {
 
         if (Objects.nonNull(rowView.getColumn("dataset_data_use", String.class))) {
           String duStr = rowView.getColumn("dataset_data_use", String.class);
-          Optional<DataUse> du = DataUse.parseDataUse(duStr);
+          Optional<DataUse> du = Optional.ofNullable(dataUseParser.parseDataUse(duStr));
           du.ifPresent(dataset::setDataUse);
```

Comment:
Is this check necessary? if the result is `null`, what is the harm of calling `setDataUse`?
```
          dataset.setDataUse(dataUseParser.parseDataUse(duStr));
```

Reply:
Great point, I think this was an artifact of the original return signature which was an optional - definitely not necessary any longer.